aderyn_core/src/detect/experimental/immediate_parent.rs

```diff
@@ -31,15 +31,15 @@ impl IssueDetector for ImmediateParentDemonstrator {
             if let Some(first_parent) = assignment.parent(context) {
                 if let ASTNode::Block(block) = first_parent {
                     println!("1 {}", block);
-                    capture!(self, context, block);
+                    capture!(self, context, first_parent);
```

Comment:
Here, you can have `capture!(self, context, block)` as well as `capture!(self, context, first_parent)`

Reply:
this is EPIC dude. Love it!